src/lib/logs/log_self_monitoring.py

```diff
@@ -228,6 +230,21 @@ def create_self_monitoring_time_series(sfm: LogSelfMonitoring, context: LogsSfmC
                     "value": {"int64Value": sfm.records_with_too_long_content}
                 }]))
 
+    if sfm.publish_time_fallback_records:
+        time_series.append(
+            create_time_serie(
```

Comment:
we should add new timeseries to dynatrace-gcp-function-log-self-monitoring.json dashboard

Reply:
Good catch, I've missed it